ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -3,6 +3,7 @@ import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import BigNumber from 'bignumber.js';
+import Box from '../../ui/box';
```

Comment:
This component has been deprecated. Please use the component-library version. You should be able to identify deprecated components as they will show with a strike through. Let me know if you don't see this in VS Code and we can compare settings. Mine was turned off for the longest time 😅 

![Screenshot 2023-07-24 at 1 08 34 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/0778181e-0a9e-4071-ad27-f10b08689a58)

```suggestion
import { Box } from '../../component-library';
```